[release/3.1] Switch to consolidated Windows build images and Svc pool

dougbu opened the pull request:
- includes a fake Arcade update matching recent release/3.x changes

dougbu committed:
[release/3.1] Switch to consolidated Windows build images and Svc pool
 - incomplete w/o an Arcade update in this branch

MattGal reviewed (approved):
Given the challenges with Arcade 3.1, It's OK to update the eng/common folder too; the arcade update will just bring the same changes when we unblock it.

dougbu committed:
Switch to consolidated Windows build images under eng/common/
- will be overwritten by next Arcade update w/ same changes

dougbu commented:
Now includes a fake Arcade update that matches recent release/3.x changes. We don't need `windows.vs2019.amd64` in our parts of the build but Arcade apparently does.

dougbu committed:
Double the Kestrel `DefaultTimeout`
- `AppCanHandleClientAbortingConnectionMidRequest(...)` timing out fairly consistently on macOS

dougbu commented on src/Servers/Kestrel/shared/test/TestConstants.cs:
```diff
@@ -8,6 +8,6 @@ namespace Microsoft.AspNetCore.Testing
     {
         public const int EOF = -4095;
-        public static TimeSpan DefaultTimeout = TimeSpan.FromSeconds(30);
+        public static TimeSpan DefaultTimeout = TimeSpan.FromMinutes(1);
```
@JamesNK @Tratcher @halter73 @BrennanConroy this seemed painless to me and is definitely the simplest option. But please let me know if you object to such a general change for what's mostly a single test failing.

JamesNK replied:
My preference is to increase the timeout on the individual test. But if that isn't possible then I don't have a problem with this.

Perhaps add a comment to say what test the 1 minute timeout is for?

dougbu replied:
I was just saving myself some copying and pasting from `TaskTimeoutExtensions` and `TestConstants`. Given your preference, I'll switch to the focused approach after dinner or tomorrow morning.

dougbu commented:
Reset approvals to remind myself I want to hear from @JamesNK @Tratcher @halter73 @BrennanConroy

dougbu committed:
Focus timeout change on `AppCanHandleClientAbortingConnectionMidRequest(...)`
